### DISCUSSION TITLE
Créer des groupes de guilde

### DISCUSSION THREAD

**Discussion by Pietagorh** [Crownicles/Crownicles#643] [Suggestions fermées]
Actuellement, il y a pas mal de rp autour de grandes guildes (aussi appelées "empires" même si c'est pas forcément vrai), mais rien d'officiel, et ce serait bien de rajouter ce côté
Je propose de rajouter :

- Des groupes de guildes (on peut les appeler nations, groupes, ou quoi, comme vous voulez), avec la possibilité de mettre plusieurs guildes sous un même chef (ça aurait un prix bien sûr)

Il y aurait des possibilités comme le transfert de pet gratuit entre les membres, voire même le transfert des membres d'une guilde à l'autre

**Comment by niqore** [comment]
Si ça se fait je propose alliance pour le nom

**Reply by Pietagorh** [reply]
Bah y'a déjà une suggestion par rapport aux alliances, mais c'est pas vraiment la même chose

**Reply by QQtin3** [reply]
Au final il y aura soit alliance soit empire, y'aura sûrement pas les deux.
Après tout est possible mais j'en doute fortement

**Reply by ghost** [reply]
Alliance ça sonne trop WoW
